File: CPP/Algoritmo1/ExercicioCondicional/exercicioCondicional03.cpp
#include <iostream>
using namespace std;
int main() {
  float nota1, nota2, media;
  int presenca, aula, frequencia;
  cout << "Informe a primeira nota do aluno: \n";
  cin >> nota1;
  cout << "Informe a segunda nota do aluno: \n";
  cin >> nota2;
  cout << "Informe quantas aulas houve: \n";
  cin >> aula;
  cout << "Informe quantas aulas o aluno compareceu: \n";
  cin >> presenca;
  media = (nota1 + nota2)/2;
  if (presenca >= aula * 0.75){
    if (media >= 6){
      cout << "Aluno esta aprovado!";
    }else{
      cout << "Aluno nao alcancou a media minima para ser aprovado";
    }
  }else{
    cout << "Aluno nao tem a frequencia minima para ser aprovado";
  }
}

File: Python/Fundamentos/exercicioCondicional03.py
ativo = True
novaNota = True
reprovado = False
nota = []
contador = 0
media = 0
while ativo == True:
    nome = input("Informe o nome do aluno ")
    if nome == "sair":
        ativo = False
    else:
        while novaNota == True and contador < 3:
            nota.append(float(input("Informe a nota ")))
            if nota[contador] < 0:
                novaNota = False
            else:
                media += nota[contador]
                contador += 1
        media = media / contador
        aulas = int(input("Informe quantas aulas houve nessa matéria "))
        presenca = int(input("Informe quantas aulas o aluno assistiu "))
        frequencia = float(aulas * 0.75)
        if aulas < frequencia:
            reprovado = True
        if media < 6:
            reprovado = True
        if reprovado == False:
            print("O aluno %s teve a média de %f e está aprovado" % (nome, media))
        else:
            print("O aluno %s teve a média de %f e está reprovado" % (nome, media))